test/expectations/inline-source-map.js

```diff
@@ -0,0 +1,11 @@
+"format register";
+
+(function() {
+function define(){};  define.amd = {};
+System.register("tree/amd-2", [], false, function() {
+  return {amd: '2'};
+});
+
+
+})();
+//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbInRyZWUvYW1kLTIuanMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7T0FPTyxFQUFFLEdBQUUsQ0FBRyxJQUFFLENBQUU7RUFBRTtBQUFBIn0=
```

Comment:
I'm worried about all these exact source tests - everytime we update compiler implementations we will be needing to update and generate these tests. How do we update the generated code? etc etc.